doc/how-to-contribute.rst

```diff
@@ -56,3 +56,94 @@ favourite tool easily.
 .. _Sphinx: http://www.sphinx-doc.org/en/stable
 .. _PyLint: https://www.pylint.org
 .. _Codacy: https://www.codacy.com/app/tango-controls/pytango/dashboard
+
+Releasing a new version
+-----------------------
+
+From time to time a new version should be released.  The basic steps required are
+as follows:
```

Comment:
Maybe advertise that anyone who wishes to see some features of the development branch released is free to make a new release themselves?